tests/test_rbuser_tostring.py

```diff
@@ -0,0 +1,80 @@
+"""RedBrick Test Module; Tests the to_string method of the rbuser module."""
+
+import unittest
+from useradm import rbuser
+
+
+class RBUserTestCase(unittest.TestCase):
+    """Test Case class for to_string method"""
+
+    def setUp(self):
+        """Call before every test case."""
+        self.billy_attr = {'cn': 'Billy Bo',
+                           'id': 15358462,
+                           'uid': 'billyx',
+                           'bday': 12,
+                           'host': 'carbon',
+                           'year': 3,
+                           'gecos': 'new member',
+                           'byear': 2017,
+                           'bmonth': 2,
+                           'passwd': 'e1NTSEF9WlBURGtJdmZJSld0WUlrak9zdGxscXYySTRRVEhBYj=',
+                           'newbie': True,
+                           'course': 'CASE',
+                           'created': '2009-10-06 11:20:13',
+                           'altmail': 'billpbo9@mailx.dcu.net',
+                           'updated': '2015-09-28 01:42:21',
+                           'birthday': '12 02 2017',
+                           'usertype': 'member',
+                           'updatedby': 'greenday',
```

Comment:
yes if we decide to change rules we will need to modify test to those rules and thats fine

also  blocking numbers would be insane. block symbols yes numbers no